### PR TITLE
pt-BR

### PR DESCRIPTION
**Pull request by j1m1l0k0**
Translation Brazilian Language

### CODE REVIEW

**Comment by GeopJr** [comment]
cc: @Yimura 

```
errors: [
    'pt_BR.json[CMD_EXECUTOR_CMD_TEMPLATE]: incorrect number of `{}` found (4:3)',
    'pt_BR.json[NOTIFICATION_WELCOME_TEXT]: incorrect number of `{}` found (1:0)',
    'pt_BR.json[PLAYER_INFO_LANGUAGE]: incorrect number of `%` found (1:0)',
    'pt_BR.json[SESSION_BROWSER_LANGUAGE]: incorrect number of `%` found (1:0)'
  ],
```

File: pt_BR.json [hunk 1]
```diff
@@ -0,0 +1,772 @@
...
+  "NEON_LIGHT_OPTIONS": "Opções de luz neon",
+  "NETWORK": "Rede",
+  "NETWORK_PLAYER_MGR": "Gestor de jogadores de rede",
+  "NETWORK_PLAYER_MGR_DESTROY": "Saindo da sessão e limpando os dados do jogador.",
+  "NETWORK_PLAYER_MGR_INIT": "Entrando na sessão e inicializando os dados do jogador.",
+  "NEVER_WANTED": "Nunca Quis",
+  "NEW_ENTRY": "Nova entrada",
+  "NO": "Não",
+  "NORMAL": "Normal",
+  "NOTIFICATION_WELCOME_TEXT": "Carregado YimMenu. ",
```
Comment (Yimura) [inline pt_BR.json:392]: Missing format argument in `NOTIFICATION_WELCOME_TEXT` for which key to press to open YimMenu.
(Original English `Loaded YimMenu. Press {} to open`)

File: pt_BR.json [hunk 2]
```diff
@@ -0,0 +1,772 @@
...
+  "SELF": "Auto",
+  "SEMI_GODMODE": "Semi Godmode",
+  "SEND": "Enviar",
+  "SEND_MSG": "Enviar mensagem",
+  "SEND_SMS": "Enviar SMS",
+  "SEND_TO_CUTSCENE": "Enviar para Cutscene",
+  "SEND_TO_LOCATION": "Enviar para o local",
+  "SESSION_BROWSER_DISCRIMINATOR": "Discriminador: 0x%X",
+  "SESSION_BROWSER_HOST_RID": "ID da Rockstar do anfitrião: %d",
+  "SESSION_BROWSER_LANGUAGE": "Línguas",
```
[the remark this answers is not in the document]
Comment (j1m1l0k0) [reply pt_BR.json:573]: making the correction.

**Comment by Yimura** [comment]
Closing in favour of #23

**Pull request closed without merging**